spec/support/features/user_login.rb

```diff
@@ -0,0 +1,9 @@
+def login_user(email, password)
```

Comment:
Unused method argument - `password`. If it's necessary, use `_` or `_password` as an argument name to indicate that it won't be used.